functions/messages.js

```diff
@@ -95,6 +98,8 @@ exports.handler = async (event) => {
       status: 200
     });
   } catch (error) {
+    await reportError(error);
```

Comment:
@arathjz I don't think is a good idea to await, because we are blocking the response. Naybe we can return the response first and then report the error?

Reply:
@orlando I added that await because of this section of the article you shared with me https://httptoolkit.tech/blog/netlify-function-error-reporting-with-sentry/#reliable-reporting-with-sentry--aws-lambda

What are your thoughts?
![image](https://user-images.githubusercontent.com/20747535/115937581-f4265a00-a45d-11eb-88f1-aed00b73c28d.png)